leadership.yaml

```diff
@@ -0,0 +1,119 @@
+- type: house
+  title: Speaker the House
+  office: Speaker
+  bioguide: B000589
```

Comment:
I'd love others' opinions using the data, but I think it's better to keep the bioguide out of this file, and rely on the identification of the leader in `legislators-current.yaml`. I say that partly over redundancy, but mostly without the `bioguide` (and my impending comment about URLs), this file can stay mostly static, instead of needing to change with shifts in people.